composer.json

```
@@ -35,7 +35,7 @@
         "friendsofsymfony/http-cache-bundle": "^2.8",
         "handcraftedinthealps/zendsearch": "^2.0",
         "jackalope/jackalope-doctrine-dbal": "^1.3",
-        "sulu/sulu": "~2.1.9",
+        "sulu/sulu": "dev-feature/link-dependencies as 2.1.10",
```

Should be reverted to:

```suggestion
        "sulu/sulu": "~2.1.9",
```